x-pack/plugins/cases/server/authorization/index.ts

```diff
@@ -241,31 +258,31 @@ export const Operations: Record<ReadOperations | WriteOperations, OperationDetai
     docType: 'comments',
     savedObjectType: CASE_COMMENT_SAVED_OBJECT,
   },
-  [ReadOperations.GetComment]: {
-    ecsType: EVENT_TYPES.access,
-    name: ACCESS_COMMENT_OPERATION,
-    action: 'case_comment_get',
-    verbs: accessVerbs,
-    docType: 'comments',
-    savedObjectType: CASE_COMMENT_SAVED_OBJECT,
-  },
-  [ReadOperations.GetAllComments]: {
+};
+
+/**
+ * Definition of all APIs within the cases backend.
+ */
+export const Operations: Record<ReadOperations | WriteOperations, OperationDetails> = {
+  ...CaseOperations,
+  ...ConfigurationOperations,
+  ...AttachmentOperations,
+  [ReadOperations.GetTags]: {
     ecsType: EVENT_TYPES.access,
-    name: ACCESS_COMMENT_OPERATION,
-    action: 'case_comment_get_all',
+    name: ReadOperations.GetTags,
```

Comment:
This is a change here. I think I had copy/pasted this as `ReadOperations.GetCase` accidentally. We actually have a `getTags` string that can be used in this file: https://github.com/elastic/kibana/blob/main/x-pack/plugins/security/server/authorization/privileges/feature_privilege_builder/cases.ts#L18

So I figured it'd be best to start using that instead.